README.md

```diff
@@ -119,39 +132,33 @@ The database seed contains most of the custom configuration, theme selection, ad
 
 ### 7. Set Magento to `production` mode
 
-The explicit call to `setup:static-content:deploy` is required due to requiring multiple locales.  `en_AU` is used in
-the frontend, and `en_US` is used for the backend admin UI.  The final command sets Magento to production mode, 
-including compiling the dependency injections and other configuration.
+In any deployed environment, Magento should be in `production` mode, as opposed to `default` or `developer`.  
 
 ```
-bin/magento setup:static-content:deploy en_AU en_US
```

Comment:
Read comment above about locale configuration (starts line 53 of new version).

Or, just try it, staging does not have an en_US locale: staging-shop.histwest.org.au/rwahs_admin?locale=en_AU

If you omit the query parameter, it breaks, but that's because you've used the wrong URL.  Once you're logged in, your account is set to en_AU so it's no longer an issue.